### PR TITLE
Correct units

### LINKED ISSUE #262: acs monitor is in Hz instead of MHz
Wow my processor is really slow lol

### PR DESCRIPTION
**Pull request by Shuzhengz**
Just realized that acs monitor prints out Hz instead of MHz
Closes #262 #135

### CODE REVIEW

File: src/display.rs [hunk 1]
```diff
@@ -114,21 +114,25 @@ pub fn print_cpu(cpu: &CPU) {
 
 pub fn render_cpu(cpu: &CPU) -> String {
     let temp: colored::ColoredString;
+    // let mut hmm;
```
Comment (JakeRoggenbuck) [inline src/display.rs:117]: probably remove this, other than that, this looks hella good!

**Comment by JakeRoggenbuck** [comment]
Also, sorry I didn't see this until now

**Comment by JakeRoggenbuck** [comment]
Thanks so much for fixing this issue

File: src/display.rs (hunk 1 above)
Comment (Shuzhengz) [reply src/display.rs:117]: oops, that was me testing if it actually worked and i forgot to delete it lol

**Comment by Shuzhengz** [comment]
yee np

**Comment by Shuzhengz** [comment]
am j gonna merge this

**Pull request merged**